manifests/params.pp

```diff
@@ -791,4 +791,10 @@
   } else {
     $verify_command = '/usr/sbin/apachectl -t'
   }
+
+  if ($facts['os']['family'] == 'RedHat' and ($facts['os']['release']['major'] >= '8') ) {
+    $ssl_protocol = ['all'] # Implementations of the SSLv2 and SSLv3 protocol versions have been removed from OpenSSL (and hence mod_ssl) because these are no longer considered secure. For additional documentation https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/deploying_different_types_of_servers/setting-apache-web-server_deploying-different-types-of-servers
+  } else {
+    $ssl_protocol = [ 'all', '-SSLv2', '-SSLv3' ]
+  }
```

Comment:
Since this module has a dependency on Puppet `>= 5.5.10` I'd love for new parameters to be added to Hiera files instead of using `params.pp`.

I'm not sure what your policy is, so it might be out of the question, but it seems a bit silly to continue using the deprecated `params.pp` style defaults when you can use Hiera instead. I'd even prefer to keep `params.pp` around (until parameters can be migrated to Hiera), and use Hiera for any new parameters, even if it means that both systems will be used for a while.

Reply:
...and as per https://puppet.com/docs/puppet/latest/hiera_migrate.html#adding_hiera_data_to_a_module using Hiera is the preferred way:

> Modules need default values for their class parameters. Before, the preferred way to do this was the “params.pp” pattern. With Hiera 5, you can use the “data in modules” approach instead.

Reply:
Again this is just to match the existing manifests, and updating to use hiera module data should probably be a separate PR to change this for all params at once.

Reply:
Yes, I understand that. I'd simply prefer to start moving parameters to Hiera *now*, instead of doing it at `Optional[String] $future_date` -- Hiera is just so much easier to comprehend and maintain than `params.pp` :smile: